langchain/vectorstores/base.py

```diff
@@ -116,6 +117,15 @@ def similarity_search_with_relevance_scores(
         """Return docs and relevance scores in the range [0, 1].
 
         0 is dissimilar, 1 is most similar.
+
+        Args:
+            query: input text
+            k: Number of Documents to return. Defaults to 4.
+            score_threshold: Optional, a floating point value between 0 to 1 to
```

Comment:
Could we expand kwargs to include this variable with a default of None?

Are we able to propagate this into the implementation itself? If there are use cases where `k` is large (say 10_000) it's nice to be able to have the data pre-filtered prior to being transmitted to the client.